lockfile/src/javascript.rs

```diff
@@ -53,6 +53,10 @@ impl Parse for PackageLock {
                 // Since we care more about the name of a local dependency than its package, we
                 // discard the version here and include the package later when it's mentioned by
                 // name.
+                if !name.starts_with("node_modules/") {
+                    continue;
+                }
+
                 let name = match name.rsplit_once("node_modules/") {
                     Some((_, name)) => name,
                     None => continue,
```

Comment:
This is technically unreachable now, but `continue` is harmless and less scary that `.unwrap()` or `unreachable!()`